src/commands/game_options/timer.ts

```diff
@@ -24,8 +26,11 @@ const logger = new IPCLogger("guessTimeout");
 
 // eslint-disable-next-line import/no-unused-modules
 export default class GuessTimeoutCommand implements BaseCommand {
-    static TIMER_MIN_VALUE = 2;
+    static TIMER_MIN_VALUE = 0.75;
```

Comment:
So this is the min value for CLIP?
Maybe call it TIMER_MIN_VALUE_CLIP, and have another variable
TIMER_MIN_VALUE = min(TIMER_MIN_VALUE_CLIP, TIMER_MIN_VALUE_NON_CLIP) to be more explicit